injects/logger.js

```diff
@@ -66,7 +66,9 @@ hook.enable()
 // before process exits, flush the encoded data to the sample file
 process.once('beforeExit', function () {
   hook.disable()
-  encoder.end()
+  encoder.end(() => {
+    out.destroy()
```

Comment:
This should not be needed, there might be a bug in the encoder stream implementation.

Anyway this is ok for now.. maybe add a comment?